index.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace NYPL\Services;
+
+use NYPL\Starter\Service;
+use NYPL\Starter\Config;
+use NYPL\Starter\ErrorHandler;
+use NYPL\Services\Controller\HoldRequestController;
+use NYPL\Services\Controller\HoldRequestResultController;
+
+require __DIR__ . '/vendor/autoload.php';
+
+try {
+    Config::initialize(__DIR__);
+
+    $container = new ServiceContainer();
+
+    $service = new Service($container);
+
+    $service->get('/docs', Swagger::class);
+
+    $service->post('/api/v0.2/hold-requests', HoldRequestController::class . ':createHoldRequest');
```

Comment:
Since we're going to "swap" out the Alpha service with this service, I wonder if we should keep the API version numbers the same? Also, it really is the same "version" from a consumer POV.